prow/pubsub/subscriber/subscriber.go

```diff
@@ -169,37 +169,37 @@ func (peh *periodicJobHandler) getProwJobSpec(cfg prowCfgClient, pc *config.InRe
 		}
 	}
 	if periodicJob == nil {
-		return nil, nil, fmt.Errorf("failed to find associated periodic job %q", pe.Name)
+		return nil, nil, nil, fmt.Errorf("failed to find associated periodic job %q", pe.Name)
```

Comment:
Nit: At some point we should stop adding more return values here and just return a struct.